plugins/outputs/azure_data_explorer/azure_data_explorer_test.go

```diff
@@ -17,6 +14,7 @@ import (
 	"github.com/stretchr/testify/require"
 
 	"github.com/influxdata/telegraf"
+	adx_common "github.com/influxdata/telegraf/plugins/common/adx"
```

Comment:
```suggestion
	common_adx "github.com/influxdata/telegraf/plugins/common/adx"
```
or remove the alias altogether if there is no collision.

Reply:
This will help us differentiate between the adx sdk and adx commons

Reply:
Yeah but we use `common_...` in other plugins so `common_adx` instead of `adx_common`.